File: Programmers/코딩테스트 고득점 Kit/해시/Lv1_완주하지 못한 선수.py
def solution(participant, completion):
    answer = {}
    for p in participant:
        if p not in answer:
            answer[p] = 0
        if p in answer:
            answer[p] += 1
    for p in completion:
        answer[p] -= 1
    for p in participant:
        if answer[p] == 1:
            return p

File: Programmers/코딩테스트 고득점 Kit/해시/Lv1_완주하지못한선수.js
function solution(participant, completion) {
  const answer = {};
  participant.forEach((p) => {
    if (!answer[p]) answer[p] = 0;
    answer[p] += 1;
  });
  completion.forEach((p) => (answer[p] -= 1));

  for (let p in answer) {
    if (answer[p] === 1) return p;
  }
}
